packages/suite/src/components/wallet/AccountsMenu/AccountGroup.tsx

```diff
@@ -74,7 +74,8 @@ export const AccountGroup = forwardRef(
         }
 
         const onClick = () => {
-            setIsOpen(previous => !previous);
+            setIsOpen(!isOpen);
```

Comment:
@komret  there is no need to change it here, we could keep `setIsOpen(previous => !previous);` but I found it more confusing when we use the `isOpen` state one line below